### DISCUSSION TITLE
Show "Crate of the week" on the frontpage

### DISCUSSION THREAD

**Discussion by Turbo87** [rust-lang/crates.io#4280] [Feature Requests]
The crates.io frontpage currently shows several lists of crates and versions, but not all of them are particularly relevant.

https://this-week-in-rust.org has a "Crate of the week" section, which regularly highlights interesting crates.

We could consider build on top of their curation and display the "Crate of the week" also on the crates.io frontpage.

Stretch goal: We could import the past "Crate of the week" data into the database and show an indicator or even the full text when a crate has been covered at some time in the past.

/cc @llogiq

**Comment by llogiq** [comment]
A few notes for the record:

* Not all crates of the week are on crates.io, or library crates or even crates at all (we once had a service that finds rust methods by type signature)
* We sometimes have zero or two crates
* Currently any crate has a name, link, description, nominator name, link to the nomination.
* I have a script to push the PR to TWiR, so I could easily extend this to either add another file, send a PR somewhere else or just upload the info, whatever suits you best.
